File: python/practices/financial_calculator.py
# HM financial calculator
monthly_income = input("What is your monthly income? $")
monthly_rent = input("How much is the rent monthly: $")
monthly_utilities = input("How much is the utilities monthly: $")
month_groceries = input("How much is the groceries monthly: $")
monthly_transportation = input("How much is monthly transportation: $")

savings = monthly_income * 0.1
money = monthly_income - monthly_rent - monthly_utilities - month_groceries - monthly_transportation - savings

print(f"Your rent is $ {monthly_rent} and that is 40 % of your income")
print(f"Your utilities is $ {monthly_utilities} and that is 7 % of your income")
print(f"Your grocieries is $ {month_groceries} and that is 7 % of your income")

File: C/practices/financial_calculator.c
// HM P7 financial calculator
#include <stdio.h>
int main(){
    int income;
    int rent;
    int utilities;
    int groceries;
    int transportation;

    printf("Your monthly income will be: \n");
    scanf("%f", &income);
    printf("Rent is: \n");
    scanf("%f", &rent);
   
    printf("Rent is %f of your income\n", (float) income/rent);
    printf("utilities is: \n");
    scanf("%f", &utilities);

    printf("Utilities is %f of your income\n", (float) income/rent);

    printf("Groceries is: \n");
    scanf("%f", &groceries);

    printf("Groceries is %f of your income\n", (float) income/rent);

    printf("Transportation is: \n");
    scanf("%f", &transportation);
    printf("Transportation is %f of your income\n", (float) income/transportation);
    
   

    return 0;

}